lib/config/presets/internal/monorepo.ts

```diff
@@ -19,6 +19,8 @@ const repoGroups = {
   angularjs: 'https://github.com/angular/angular.js',
   'apollo-server': 'https://github.com/apollographql/apollo-server',
   apolloclient: 'https://github.com/apollographql/apollo-client',
+  'applicationinsights-dotnet':
```

Comment:
i think we should use `dotnet` as prefix, as we've already a lot monorepos with that schema

Reply:
> i think we should use `dotnet` as prefix, as we've already a lot monorepos with that schema

There are two items that are prefixed with `dotnet`, and two are suffixed.
Ruby is only suffixed (but just once).
Javascript also, only just suffixed.

Reply:
🤔 Ok, we should unify those, probably always use suffix.

It should be possible to do it without breaking with using `migratePresets` feature.

@HonkingGoose Can you please open an issue to trsack that?

Reply:
> @HonkingGoose Can you please open an issue to track that?

Done, see:

- #16766

Reply:
As the plan seems to be using a suffix in the future and this PR already uses a suffix, I marked this as resolved.